### DISCUSSION TITLE
how to access static field?

### DISCUSSION THREAD

**Discussion by encoderlee** [vfsfitvnm/frida-il2cpp-bridge#74] [Q&A]
![image](https://user-images.githubusercontent.com/9878012/147931645-c02bba0a-e6b2-4cdc-9df1-52741264b367.png)

![image](https://user-images.githubusercontent.com/9878012/147931928-c8714929-f0c8-40ee-bfc2-ca4522b70549.png)

how to get static field 'gameState' value?
 thanks you very much !

**Comment by vfsfitvnm** [comment]
You get the `Global` `Il2Cpp.Class`, then use the property accessor `fields.gameState` to get a `Il2Cpp.Field`:
```ts
Il2Cpp.perform(() => {
    const AssemblyCSharp = Il2Cpp.Domain.assemblies["Assembly-CSharp"].image;
    const Global = AssemblyCSharp.classes.Global;
    const gameState = Global.fields.gameState;
    const gameStateValue = gameState.value as Il2Cpp.ValueType;
    
    console.log(`gameStateValue: ${gameStateValue}`);
});
```

**Reply by encoderlee** [reply]
thanks
